connexion/jsonifier.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import json
+import uuid
+
+import six
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
```

Comment:
If i'm reading the `codacy` output correctly, I think you might need to rename this method, and then set it as an attribute named `default`.

Reply:
Is there a possibility to mark this a false positive for Codacity ?
Because it's the standard way to extend JSONEncoder. If we hack our code for Codacity, it can confuse other developers which would like to extend our own JSONEncoder.

Reply:
You are right. There's an example from the python docs that does it exactly as you did: https://docs.python.org/2/library/json.html

I think it's fine to ignore it then.